Build Failure - Task :checkLicenses FAILED

<!-- Have you done the following? -->
<!--   * read the Code of Conduct? By filing an Issue, you are expected to -->  
<!--     comply with it, including treating everyone with respect: -->
<!--     https://github.com/hyperledger/besu/blob/master/CODE_OF_CONDUCT.md -->
<!--   * Reproduced the issue in the latest version of the software -->
<!--   * Read the debugging docs: https://besu.hyperledger.org/en/stable/HowTo/Monitor/Logging/ -->
<!--   * Duplicate Issue check:  https://github.com/search?q=+is%3Aissue+repo%3Ahyperledger/Besu -->
<!-- Note:  Not all sections will apply to all issue types. -->

### Description
As an developer, I want to be able to build master without any build script errors. 

### Steps to Reproduce (Bug)
1. run ./gradlew clean spotlessApply build

**Expected behavior:** Should be able to run the build without any issues

**Actual behavior:** The first time every time it fails with the following error
`
./gradlew clean spotlessApply build
Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details
> Task :checkLicenses FAILED

FAILURE: Build failed with an exception.

* Where:
Script '/home/perusworld/works/open/contrib/besu/gradle/check-licenses.gradle' line: 166

* What went wrong:
Execution failed for task ':checkLicenses'.
> java.io.FileNotFoundException: /home/perusworld/works/open/contrib/besu/build/reports/license/license-dependency.xml (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 39s
318 actionable tasks: 318 executed
`

**Frequency:** It happens the first time everytime and if i run the same command again, it runs without any issue

### Versions (Add all that apply)
* Java version: [openjdk 11.0.10 2021-01-19]
* OS Name & Version: [Ubuntu 20.04.2 LTS]
* Kernel Version: [5.8.0-48-generic]

### Additional Information

--- a/gradle/check-licenses.gradle
+++ b/gradle/check-licenses.gradle
@@ -153,7 +153,7 @@ downloadLicenses {
 }
 
 task lazyDownloadLicenses() {
-  if (!file("$rootProject.buildDir/reports/license/license-dependency.xml").exists()) {
+  if (gradle.startParameter.taskNames.contains('clean') || !file("$rootProject.buildDir/reports/license/license-dependency.xml").exists()) {
     dependsOn ':downloadLicenses'
   }
 }

